packages/hub/prisma/migrations/20230823211259_squiggle_version/migration.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE "SquiggleSnippet" ADD COLUMN     "version" TEXT;
+UPDATE "SquiggleSnippet" SET "version" = '0.8.5' WHERE "version" IS NULL;
```

Comment:
That seems weird. If we release new functionality, people won't be able to use it  unless they upgrade, which we don't have a route for now.

Can we, for now, just save this data, but also just have that not effect things in the Squiggle Hub? We can work on upgrading / automatic stuff later on, I assume that will be more work+thinking.

My impression before was that this PR was mainly for the Website/Playground, not for trying to secure this with the Hub.

Reply:
I definitely want it on Squiggle Hub, i.e. if someone stores a model there, there should be a guarantee that the model won't break because we updated something.

Applying `UPDATE "SquiggleSnippet" SET "version" = '0.8.6' WHERE "version" IS '0.8.5';` on a new release is easy, so we could always do that if we're confident that the release is backward-compatible. But I think pinning should be the default outcome.

Users could also upgrade by themselves, by opening a model, changing its version and saving.

Reply:
Also, there's still a way to opt-in to "please auto upgrade my models, I want to be on a bleeding edge", by picking `dev` version. But I still think that pinning the latest available version is a better default.

(Btw, I also want to pin seed when we finally get around to #227, then we could have full reproducibility guarantees)

Reply:
How about we work to figure this part out next time we have real breaking changes? Until then, we could just save ``version`` - but show the most recent version of the playground.

Reply:
I'm still not convinced (for clarity, I didn't propose to do codemods on breaking changes, just to keep an old version untouched), but ok.

What's the most recent version then? `dev` -> `dev`, anything else -> latest stable?